### PR TITLE
Use workspace version for members

### PR DESCRIPTION
**Pull request by guilload**
### How was this PR tested?
`make test-all`

### CODE REVIEW

File: quickwit/quickwit-cli/Cargo.toml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-documentation = "https://quickwit.io/docs/"
-default-run = "quickwit"
+
+version.workspace = true
+edition.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+repository.workspace = true
+authors.workspace = true
+license.workspace = true
```
Comment (trinity-1686a) [inline quickwit/quickwit-cli/Cargo.toml:10]: maybe we want to keep `default-run = "quickwit"`?

**Review by trinity-1686a — APPROVED** [review]
I'm unsure if we want to remove `descriptions =` from all crates

File: quickwit/quickwit-cli/Cargo.toml (hunk 1 above)
Comment (guilload) [reply quickwit/quickwit-cli/Cargo.toml:10]: Good catch. I didn't notice that one.

**Comment by guilload** [comment]
I will put the descriptions back and rewrite the useless ones.

**Pull request merged**